framework/doc/content/source/auxkernels/ReactionForceAux.md

```diff
@@ -0,0 +1,14 @@
+# ReactionForceAux
+
+The value of a tagged vector for a given variable is coupled to
+the current AuxVariable. ReactionForceAux return the unscaled coupled DOF value with units consistent with the simulation.
+AuxVariable then is written out in an exodus file.
+
+!warning
+This AuxKernel assumes that the coupled tagged vector is affected by variable scaling. This assumption holds for the residual vector, but does not for the solution vector.
```

Comment:
right so since we know what the solution vector tag is, we could warn the user in the code